### ISSUE TITLE
TZUNTIL empty with latest version

### ISSUE THREAD

**Issue by bobvandevijver** [iCalcreator/iCalcreator#101]
It looks like the latest releases (2.41.39 & 2.41.43) cleared the  `TZUNTIL` value from our iCal export. Is this to be expected?

```
1) App\Tests\Service\User\UserIcalServiceTest::testCustomEventCalendar
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
 X-LIC-LOCATION:Europe/Amsterdam\n
 BEGIN:VTIMEZONE\n
 TZID:Europe/Amsterdam\n
-TZUNTIL:20231029T010000Z\n
+TZUNTIL:\n
 BEGIN:STANDARD\n
 TZNAME:CET\n
 DTSTART:20211031T030000\n
```

And if it is empty, shouldn't it just be removed as it is an optional property?

**Comment by iCalcreator** [comment]
No, not expected, it is a TZUNTIL (trait) bug, fixed in next release after tests

**Comment by iCalcreator** [comment]
Fixed in the new release 2.41.44

**Comment by bobvandevijver** [comment]
I can confirm it is indeed fixed, thank you.
